### ISSUE TITLE
🐞 Duplicate emails should not be allowed

### ISSUE THREAD

**Issue by jontyk** [openline-ai/openline-customer-os#536]
**Describe the bug**
It is possible to create duplicate emails (see current dev environment). This should not be possible and should error correctly.

**Expected behavior**
Email should be unique per email per tenant (i.e. bob@krispykreme.com should be able to exist against multiple tenants, but only once)

Thank you for your bug report!  We ❤️ squashing 🪲

**Comment by edifirut** [comment]
duplicate of https://github.com/openline-ai/openline-customer-os/issues/177
